docs/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [1.0.12] (April 2024)
```

Comment:
i have included this in the [postgres user password fix](https://github.com/ionos-cloud/crossplane-provider-ionoscloud/pull/224) so I think we can close this pr